CONTRIBUTING.md

```diff
@@ -31,8 +31,15 @@ For running the tests you will need:
  - pytest-benchmark
  
  
- All should be pip installable.
+All should be pip installable.
 
+```python -m pip install sphinx sphinx_rtd_theme pytest pytest-cov pytest-benchmark```
+
+Venv:
++ If developing a [virtual environment](https://docs.python.org/3/library/venv.html) (recommended), use this command while in the environment to download all relevant dependencies
++ ```python -m pip install pygame python-i18n importlib_resources typing_extensions```
++ Or here's a one liner to set up the virtual environment all at once
++ ```python -m pip install pygame python-i18n importlib_resources typing_extensions sphinx sphinx_rtd_theme pytest pytest-cov pytest-benchmark ```
```

Comment:
Just a note here - you only need importlib_resources if you are on Python 3.7 or 3.8 as the resources module became part of the python standard library in 3.9. 

You also only need typing_extensions if you are on Python 3.7 as TypedDict was added to Python in 3.9.

Both of these will be dropped as dependencies when pygame drops Python 3.7 and 3.8 as I'm trying to make this library follow pygame in terms of minimum versions of python.